code/mob/living/critter/ai/sawflyai.dm

```diff
@@ -119,24 +122,23 @@
 	..()
 
 /datum/aiTask/sequence/goalbased/sawfly_chase_n_stab/get_targets()
```

Comment:
why do we have two different procs that both search for targets at range and attack them? Seems like these do the same thing but with twice the performance loss